bitnami/grafana/values.yaml

```diff
@@ -291,6 +291,7 @@ resources:
 ##
 livenessProbe:
   enabled: true
+  scheme: HTTP
```

Comment:
Hi,
  You need to add this parameter to the comment block above. This will help to autogenerate parameters sections in the README.md file.

Reply:
As a reference in order to help you, the comment should look like this:
```
  ## @param livenessProbe.scheme Scheme for livenessProbe
```
Place it next to the other `@param livenessProbe.*` and that should work.